Gems/ScriptCanvas/Code/Include/ScriptCanvas/AutoGen/ScriptCanvasAutoGenRegistry.h

```diff
@@ -6,89 +6,97 @@
  *
  */
 
-#pragma once
+ //!
+ //! Script Canvas nodes need to be registered into O3DE in order to be
+ //! available to the different systems. Because ScriptCanvas nodes are
+ //! components, this registration system allows the collecting
+ //! AZ::ComponentDescriptors in order to register them with the
+ //! application.
+ //!
+ //! The goal is to make ScriptCanvas node creation as straightforward
+ //! as possible. By creating a small helper factory that registers
+ //! nodes into an AZ::EnvironmentVariable we remove all registration
+ //! related boilerplate code, developers need simply to add their
+ //! ScriptCanvas autogen drivers, header, and source files to the
```

Comment:
Suggestion: Don't really need to talk about what's been removed, since it's no longer there. The comment doesn't make sense unless you know the history of the code.

```suggestion
 //! as possible. By creating a small helper factory that registers
 //! nodes into an AZ::EnvironmentVariable, developers need simply to add their
 //! ScriptCanvas autogen drivers, header, and source files to the
```